### PR TITLE
Event Cultist Ship/Added teh KoS Express-class Eldritch Hijacked Freighter

### PR DESCRIPTION
**Pull request by BlackCat-055**
## About The Pull Request
I made a Ship that's cultist in theme, essentially was originally an Express Cargo Freighter that got hijacked by Nar'sian cultists as a base of operations. Figured something like this could be done for events!

Name: KOS Express-class Eldritch Hijacked Freighter

scrnshot

## Why It's Good For The Game

Because funny Cultship that can be used for events to spice things up. It gives cultists for a possible cultist event a fitting headstart-ish on their Cult stuff.

## Changelog
:cl:
add: A Cultist Ship!
config: Did the cultist ship config
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
Don't merge until i add da thing to make people cultists

**Comment by Jackriip** [comment]
test merging

**Comment by Jackriip** [comment]
nevermmind you need to add that antag status json thing

**Comment by BlackCat-055** [comment]
All done now! At least from what I can see

**Comment by BlackCat-055** [comment]
I maay have to fix some things.

**Comment by BlackCat-055** [comment]
Yeah I dont know how to test ships but I think everything works

**Comment by dwasint** [comment]
it works gonna hold off on tm'ing or merging until parts works

File: _maps/configs/KOS_express_cultist.JSON [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+			"outfit": "/datum/outfit/job/engineer/hazard",
+			"slots": 1
+		},
+		"Converted Miner": {
+			"outfit": "/datum/outfit/job/miner/hazard",
+			"slots": 2
+		},
+		"Cultist": {
+			"outfit": "/datum/outfit/job/assistant",
+			"slots": 3
```
Comment (Jackriip) [inline _maps/configs/KOS_express_cultist.JSON:27]: bad json

**Review by Jackriip — CHANGES_REQUESTED** [review]

**Comment by SuperNovaa41** [comment]
It doesn't work, the json file is missing a bracket, and it has trailing commas. Also the map file doesn't have a docking port
image

**Pull request closed without merging**